File: src/examples/csharp/webhooks/createWebhook.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace csharp
{
    class createWebhook
    {
        static async Task Main()
        {
            EasyPost.ClientManager.SetCurrent(Environment.GetEnvironmentVariable("EASYPOST_TEST_API_KEY"));

            Webhook webhook = await Webhook.Create(
                new Dictionary<string, object>() {
                    { "url", "https://www.example.com" }
                }
            );
            Console.WriteLine(JsonConvert.SerializeObject(webhook, Formatting.Indented));
        }
    }
}

File: src/examples/python/webhooks/create_webhook.py
import os

import easypost
from dotenv import load_dotenv

load_dotenv()
easypost.api_key = os.getenv('EASYPOST_TEST_API_KEY')

webhook = easypost.Webhook.create(
    url="https://webhook.site/123",
    webhook_secret="123",
)

print(webhook)
